### PR TITLE
Add getFriendlyName for Android third party extensions

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**

### CODE REVIEW

File: resources/building-mobile-extensions/building-an-extension.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ All Adobe module names are prefixed with `com.adobe.module` and are considered r
 
 The `Extension` class has the following methods that you can optionally override and a member that provides access to the Event Hub:
 
+* `getFriendlyName`: Retuns a friendly name for your extension.
```
Comment (calebk1) [inline resources/building-mobile-extensions/building-an-extension.md:52]: Just to confirm, will your average user know what "a friendly name" entails?

**Review by calebk1 — COMMENTED** [review]
Looks good! Just one comment/question.

File: resources/building-mobile-extensions/building-an-extension.md (hunk 1 above)
Comment (spoorthipujariadobe) [reply resources/building-mobile-extensions/building-an-extension.md:52]: I guess so, @emdobrin do you think we should provide more details about what the friendly name is and how we're using it?

Comment (emdobrin) [reply resources/building-mobile-extensions/building-an-extension.md:52]: I think this should be sufficient. We can add something like: `Returns a friendly name for your extension, usually referenced in logs.`

**Review by emdobrin — APPROVED** [review]

**Pull request merged**